src/monitor/monitor.controller.ts

```diff
@@ -169,4 +174,39 @@ export class MonitorController {
   async getMonitorAvailability(@Param('monitorId') monitorId: string) {
     return this.monitorService.getMonitorAvailability(+monitorId);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @ApiBearerAuth()
+  @Roles(Role.Professor)
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @ApiOperation({
+    summary: 'Atualiza o status de uma monitoria para 4 (finalizada)',
+  })
+  @Patch(':id_monitoring/end')
```

Comment:
O ideal é a gente padronizar o case que a gente usa no repositório inteiro. Vamo sempre dar prioridade pro camelCase:
```suggestion
  @Patch(':monitorId/end')
```